### PR TITLE
Small improvements of porting tips

### PR DESCRIPTION
**Pull request by sudo-nice**
Please, see commits.

### CODE REVIEW

File: docs/17-porting-tips.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ you might have added to your `rc.lua`.
 This avoids most of the possible errors due to missing important changes.
 
 If you still wish to ignore this advice, first read the
-<a href="89-NEWS.md.html#v4">NEWS</a> section about the breaking changes. This
+[NEWS](89-NEWS.md) section about the breaking changes. This
```
Comment (psychon) [inline docs/17-porting-tips.md]: I'm not sure about this one. The `#v4` was added there on purpose.

Does your ikiwiki generate a different file name than `89-NEWS.md` or why does this link to be fixed?

Comment (sudo-nice) [inline docs/17-porting-tips.md]: Oh, sorry, my bad. I tried to open the link out of wiki and decided it's broken.

Comment (sudo-nice) [inline docs/17-porting-tips.md]: You can pick the right commit from the whole PR, can't you?

Comment (Elv13) [inline docs/17-porting-tips.md]: Better rebase it to remove them force push, GitHub cherry-picking GUI is limited.

Comment (sudo-nice) [inline docs/17-porting-tips.md]: I've reverted that questionable commit. Are we good now?

Comment (Elv13) [inline docs/17-porting-tips.md]: As far as I know, GitHub doesn't cancel out reverted commits, you have to remove the commits. You can do that using `git rebase -i origin/master; git push --force sudo-nice master`. In the rebase file, just remove the lines. As I told someone yesterday, I have a little script to makes this easier, but in your case, it's pretty straight forward in the first place. 
https://github.com/awesomeWM/awesome/pull/1510#issuecomment-276990321

**Comment by sudo-nice** [comment]
I've made a brand new PR from the scratch. Please, see https://github.com/awesomeWM/awesome/pull/1519

File: docs/17-porting-tips.md (hunk 1 above)
Comment (blueyed) [inline docs/17-porting-tips.md]: @Elv13 
It could have been squash-merged from the GitHub UI.

**Pull request closed without merging**